src/server/rpc/procedures/chart/chart.js

```diff
@@ -96,6 +96,12 @@ function genGnuData(lines, lineTitles, lineTypes, smoothing){
     });
 }
 
+/**
+ * draw charts and histograms
+ * @param {Array} lines a single line or list of lines. Each line should be in form of [[x1,y1], [x2,y2]]
+ * @param {Array=} options optional options
```

Comment:
The description here doesn't provide a whole lot of useful information. I would consider saying something like "Configuration for graph title, axes, and more" or something